peps/pep-0752.rst

```diff
@@ -0,0 +1,399 @@
+  maintain type stubs for various packages. The stub packages they maintain
+  mirror the package name they target and are prefixed by ``types-``. For
+  example, the package ``requests`` has a stub that users would depend on
+  called ``types-requests``.
+* Major cloud providers like Amazon, Google and Microsoft have a common prefix
+  for each feature's corresponding package [1]_. For example, most of Google's
+  packages are prefixed by ``google-cloud-`` e.g. ``google-cloud-compute`` for
+  `using virtual machines <https://cloud.google.com/products/compute>`__.
+* Many projects support a model where some packages are officially maintained
+  and third-party developers are encouraged to participate by creating their
+  own.
+
+  - `MkDocs <https://github.com/mkdocs/mkdocs>`__ is a documentation framework
+    based on Markdown files. They have the concept of
+    `plugins <https://www.mkdocs.org/dev-guide/plugins/>`__ which may be
+    developed by anyone and by convention are prefixed by ``mkdocs-``.
+  - `Project Jupyter <https://jupyter.org>`__ is devoted to the development of
+    tooling for sharing interactive documents. They support `extensions`__
+    which in most cases (and in all cases for officially maintained extensions)
+    are prefixed by ``jupyter-``.
+  - `Datadog <https://www.datadoghq.com>`__ offers observability as a service
+    for organizations at any scale. The
+    `Datadog Agent <https://docs.datadoghq.com/agent/>`__ ships out-of-the-box
+    with
+    `official integrations <https://github.com/DataDog/integrations-core>`__
+    for many products, like various databases and web servers, which are
+    distributed as Python packages that are prefixed by ``datadog-``. There is
+    support for creating `third-party integrations`__ which customers may run.
+
+__ https://jupyterlab.readthedocs.io/en/stable/user/extensions.html
+__ https://docs.datadoghq.com/developers/integrations/agent_integration/
+
+Such projects are uniquely vulnerable to attacks stemming from malicious actors
+squatting anticipated package names. For example, say a new product is released
+for which monitoring would be valuable. It would be reasonable to assume that
+Datadog would eventually support it as an official integration. It takes a
+nontrivial amount of time to deliver such an integration due to roadmap
+prioritization and the time required for implementation. It would be impossible
+to reserve the name of every potential package so in the interim an attacker
+may create a legitimate-appearing package which would execute malicious code at
+runtime. Not only are users more likely to install such packages but doing so
+taints the perception of the entire project.
+
+Namespacing also would drastically reduce the incidence of
+`typosquatting <https://en.wikipedia.org/wiki/Typosquatting>`__
+because typos would have to be in the prefix itself which is likely to be a
+short, well-known identifier like ``aws-``.
```

Comment:
I changed this text to:

```rst
because typos would have to be in the prefix itself which is
`normalized <naming_>`_ and likely to be a short, well-known identifier like
``aws-``.
```